### PR TITLE
Create the generator.

### PR DESCRIPTION
**Pull request by nedSaf**
#1

### CODE REVIEW

File: app/templates/test/tests.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+var selectedCaps = process.env.SELECTED_CAPS || undefined;
+var caps = selectedCaps ? capsConfig[selectedCaps] : undefined;
+
+var providerPrefix = process.env.PROVIDER_PREFIX ? process.env.PROVIDER_PREFIX + '-' : '';
+var testName = selectedCaps ? providerPrefix + selectedCaps : providerPrefix + 'default';
+
+var baseUrl = process.env.BASE_URL ? process.env.BASE_URL : 'https://www.drupal.org';
+
+describe('Drupal.org live site testing', function() {
```
Comment (amitaibu) [inline app/templates/test/tests.js]: => `Visual monitor testing`

**Comment by nedSaf** [comment]
Executing `yo shoov`:
generator

**Comment by nedSaf** [comment]
@amitaibu PR is ready.

**Comment by amitaibu** [comment]
`npm install`

**Comment by nedSaf** [comment]
Executing `yo shoov`:
generator

**Comment by nedSaf** [comment]
PR is ready for code review.

**Comment by amitaibu** [comment]
All the files should be under `ui-tests`

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**